yaml/ver10/cloudintegration/yaml/cloudintegration.yaml

```diff
@@ -0,0 +1,149 @@
+openapi: 3.1.0
+info:
+  title: Cloud Integration Specification
+  contact:
+    name: ONVIF Technical Specification
+    url: https://github.com/onvif/specs
+    email: help@onvif.org
+  license:
+    name: ONVIF Contributor License Agreement
+    url: https://github.com/onvif/specs/blob/development/LICENSE.md
+  version: "25.06"
+
+components:
+  schemas:
+    Base64DERencodedASN1Value:
+      description: This schema is the equivalent of tas:Base64DERencodedASN1Value in the Security Service specification.
+      type: string
+      format: byte
+      examples: [ Base64DERencodedASN1Value ]
+
+    EndpointReferenceType:
+      description: This schema is the equivalent of wsa:EndpointReferenceType in the WA-Addressing 1.0 specification.
+      type: string
+      format: uri
+
+    AuthorizationServerConfiguration:
+      type: object
+      description: This schema is the equivalent of tas:AuthorizationServerConfigurationData
+      properties:
+        ServerURI:
+          description: Authorization server address
```

Comment:
I went back to security spec to find out what this was precisely, and was still a bit confused :D I can see three things that this URL could be:
- Authority of the authorization server e.g. https://accounts.google.com/
  Where the device can find the discovery document (appending .well-known/openid-configuration)
- The discovery document URL itself e.g. https://accounts.google.com/.well-known/openid-configuration
- The actual token endpoint e.g. https://oauth2.googleapis.com/token (Taken from the discovery just above)

I'm not sure which it could be, all 3 make sense

Reply:
For me it is the equivalent of the data structure. Two fields are missing, KeyId and CertificateID because the device will define them and the OCS cannot know them a priori.